src/Core/Transformers/JsonApi/Request/requestSerializer.js

```diff
@@ -2,7 +2,18 @@
  * @param {Object} data
  * @param {Object} headers
  */
-/* eslint-disable-next-line no-unused-vars */
-export default function requestSerializer(data, headers) {
-  return data;
+import serializeCreateTask from './Serializers/serializeCreateTask';
+import serializeUpdateTask from './Serializers/serializeUpdateTask';
+import serializeRelationTask from './Serializers/serializeRelationTask';
+
+export default function requestSerializer(data) {
+  const serializers = [
+    serializeCreateTask,
```

Comment:
waar worden deze gebruikt? Zie het volgens mij nog nergens

Reply:
in de reduce onder de definitie worden ze gebruikt. Het is een soort van strategy pattern